src/Components/Paginator/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+import styles from './style.module.scss';
+
+function Paginator(props) {
+  // Need to pass in number of entries and desired entries per page
+  const { entries, entriesPerPage } = props;
+  const pages = Math.ceil(entries / entriesPerPage);
+  const pageNumbers = [];
+  const [currentPage, setCurrentPage] = useState(1);
+
+  // Below 3 lines can be reused on the page component
```

Comment:
would you remove this block of comments please? its awesome to leave notes but this looks like something that belongs on a trello card, personal notebook etc. 
comments are mostly for explaining confusing/large methods/logic-flows etc.

Reply:
✅